iRonCub-Mk3/wrappers/motorControl/left_arm-mc_wrapper.xml

```diff
@@ -3,7 +3,7 @@
 
 <device xmlns:xi="http://www.w3.org/2001/XInclude" name="left_arm-mc_nws_yarp" type="controlBoard_nws_yarp">
   <param name="name"> /icub/left_arm </param>
-  <param name="period"> 0.005   </param>
+  <param name="period"> 0.002   </param>
```

Comment:
In this group of files, I see you left the head wrapper period `0.005`

Reply:
Yes, I left it at 0.005 because also the board of the head runs at 200 Hz.